SECURITY.md

```diff
@@ -0,0 +1,45 @@
+# Security
+
+This document specifies the security process for the SwiftNIO project.
+
+## Disclosures
+
+### Private Disclosure Process
+
+The SwiftNIO core team asks that known and suspected vulnerabilities be
+privately and responsibly disclosed by emailing
+[cve@forums.swift.org](mailto:cve@forums.swift.org) with the [details usually
+included with bug reports][issue-template]. **Do not file a public issue.**
+
+#### When to report a vulnerability
+
+* You think you have discovered a potential security vulnerability in SwiftNIO
+  or any of the SwiftNIO projects.
+* You are unsure how a vulnerability affects SwiftNIO or any of the SwiftNIO
+  projects.
+
+#### What happens next?
+
+* A member of the team will acknowledge receipt of the report within 3
```

Comment:
Should we specify which country?

Reply:
Given emails currently go to core team I switched the country to the US.